app/facades/food_facade.rb

```diff
@@ -0,0 +1,9 @@
+class FoodFacade
+  def self.search_for_food(query)
+    json = FoodService.search_for_food(query)
+
+    json[:foods].map do |food_data|
+      Food.new(food_data)
+    end
+  end
```

Comment:
This method could return multiple values to the controller without hitting the API endpoint more than once. Specifically, the value in the local var `json` has an attribute called `totalHits` which could be used to display the total count of search results:
```
def self.search_for_food(query)
  json = FoodService.search_for_food(query)
  foods = json[:foods].first(10).map do |food_data|
    Food.new(food_data)
  end
  [json[:totalHits], foods]
end
```
You could then assign these values in the controller by doing something like
`@total_result_count, @results = FoodFacade.search_for_food(query)`